core/db/migrate/20201111133031_migrate_default_billing_addresses_to_address_book.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class MigrateDefaultBillingAddressesToAddressBook < ActiveRecord::Migration[5.2]
+  def up
+    adapter_type = connection.adapter_name.downcase.to_sym
```

Comment:
Thinking about this again, I think we should add a safe guard here. Since we already shipped this code maybe someone already migrated their data.

Reply:
This is true also for people that already updated to 2.11 and has some address with default_billing to true, isn't it?

Reply:
Yes. I've added a safeguard so that the migration does not do anything if any default_billing flags are already set.

Reply:
But that only avoids the migration (and the fix) to run for people that used 2.11 for some time before this will be released, isn't it? The ideal would be something that fixes their database as well. Do you think it's possible?

Reply:
@mamhoff any thoughts on this? Not sure I was clear enough, I meant that I think this will prevent the issue to happen only for users upgrading from Solidus `2.10.x` to `2.11.4` (assuming `2.11.4` will contain this fix), but doesn't cover who upgrades from `2.11.3` to `2.11.4`. Does it make sense?